backend/src/routes/report.js

```diff
@@ -12,19 +12,16 @@ const routes = [
   {
     method: "GET",
     url: `/${what}/projects/ProjectBudgetReport`,
-    schema: validators.getOne,
```

Comment:
Am i under the assumption that these validators are out of scope of the ticket or not required!